modules/calcite/README.txt

```diff
@@ -13,6 +42,9 @@ When starting a standalone node, move 'optional/ignite-calcite' folder to 'libs'
 Note: At now some logic from ignite-indexing module is reused, this means ignite-indexing module also
 has to be present at classpath.
 
+Connection url via jdbc with calcite engine: jdbc:ignite:thin://127.0.0.1:10800?queryEngine=calcite
```

Comment:
Let's add something like `If more than one engine configured ... `
Also let's add about other ways to send query to specific engine (ODBC, hints)
JDBC - should be in upper case (perhaps "Calcite", "H2" - start with upper case too).